### PR TITLE
Change msgpack's default object size to maximum

### PR DESCRIPTION
**Pull request by phoebusm**
Set msgpack's default object size to 2^32 - 1 bytes (maximum size supported by msgpack)

### CODE REVIEW

File: python/arcticdb/version_store/_normalization.py [hunk 1]
```diff
@@ -807,9 +807,9 @@ class MsgPackNormalizer(Normalizer):
     Fall back plan for the time being to store arbitrary data
     """
 
-    MMAP_DEFAULT_SIZE = 1 << 32  # Allow up to 4 gib pickles in memory, most of these compress fairly well.
+    MMAP_DEFAULT_SIZE = (1 << 32) - 1  # Allow up to 4 gib pickles in memory, most of these compress fairly well.
```
Comment (qc00) [inline python/arcticdb/version_store/_normalization.py:810]: The comment is no longer relevant. The limitation is from the library, not our design choice.

**Review by qc00 — APPROVED** [review]

File: python/arcticdb/version_store/_normalization.py (hunk 1 above)
Comment (phoebusm) [reply python/arcticdb/version_store/_normalization.py]: Comment updated.
I have considered removing the variables but doing so may break backward compatibility.

File: python/tests/conftest.py [hunk 2]
```diff
@@ -50,7 +50,6 @@
 
 BUCKET_ID = 0
 
-
```
Comment (qc00) [inline python/tests/conftest.py:53]: What's the reason for reverting this? Only doing it in `test_normalization.py` will mean most other test jobs will still fail (and the unit tests may fail before it gets to that file).

Comment (phoebusm) [reply python/tests/conftest.py:53]: The unittest on my branch fails regardless of the existence of setting of `MMAP_DEFAULT_SIZE` in `conftest.py` is. So at the moment I am fidgeting it.

**Review by qc00 — CHANGES_REQUESTED** [review]

File: python/tests/unit/arcticdb/version_store/test_normalization.py [hunk 3]
```diff
@@ -43,7 +43,7 @@
     "pd_ts": {"a": pd.Timestamp("2018-01-12 09:15"), "b": pd.Timestamp("2017-01-31", tz="America/New_York")},
 }
 
-# Use a smaller memory mapped limit for this test, no point memor mapping 2g
+# Use a smaller memory mapped limit for all tests, or will hit the memory size limit in github action
 MsgPackNormalizer.MMAP_DEFAULT_SIZE = 20 * (1 << 20)
```
[the remark this answers is not in the document]
Comment (phoebusm) [reply python/tests/unit/arcticdb/version_store/test_normalization.py:47]: Yes sorry my bad... I was going to remove it

File: python/arcticdb/version_store/_normalization.py [hunk 4]
```diff
@@ -1173,7 +1172,7 @@ def denormalize(self, item, norm_meta):
 normalize = _NORMALIZER.normalize
 denormalize = _NORMALIZER.denormalize
 
-_MAX_USER_DEFINED_META = 16 << 20
+_MAX_USER_DEFINED_META = MsgPackNormalizer.MMAP_DEFAULT_SIZE
```
[the remark this answers is not in the document]
Comment (phoebusm) [reply python/arcticdb/version_store/_normalization.py:1175]: Hmm I think this line is the purpose of this ticket

[the remark this answers is not in the document]
Comment (qc00) [reply python/arcticdb/version_store/_normalization.py:1175]: The metadata is a completely different beast

[the remark this answers is not in the document]
Comment (phoebusm) [reply python/arcticdb/version_store/_normalization.py:1175]: Oh no... I guess I am doing this in a completely wrong way...
Will delete this PR

[the remark this answers is not in the document]
Comment (qc00) [reply python/arcticdb/version_store/_normalization.py:1175]: What was the motivation for changing this value? I don't think it's in scope for this task?

**Pull request closed without merging**